src/test/java/racingcar/RacingGameTest.java

```diff
@@ -1,15 +1,46 @@
 package racingcar;
 
-import org.assertj.core.api.Assertions;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.function.Function;
+
+import org.junit.jupiter.api.Test;
 
 public class RacingGameTest {
 
-	@ParameterizedTest
-	@ValueSource(ints = {1, 2, 3})
-	void 레이싱_게임을_초기화하면_입력받은_숫자만큼_차량이_초기화된다(int num) {
-		RacingGame game = new RacingGame(num);
-		Assertions.assertThat(game.getCars().size()).isEqualTo(num);
+	@Test
+	void 레이싱_게임_시작시_유저를_입력받아_초기화한다() {
+		Function<String, RacingGame> gameInit = RacingGame::new;
+		RacingGame game = gameInit.apply("rac1,rac2,rac3");
+
+		assertThat(game.getCars().get(0).getName()).isEqualTo("rac1");
+		assertThat(game.getCars().get(1).getName()).isEqualTo("rac2");
+		assertThat(game.getCars().get(2).getName()).isEqualTo("rac3");
+	}
+
+	@Test
+	void 레이싱_전략만큼_차량을_전진시킨다() {
+		Function<String, RacingGame> gameInit = RacingGame::new;
+		RacingGame game = gameInit.apply("rac1,rac2,rac3");
+		MoveStrategy strategy = () -> new FixedStrategy().move();
```

Comment:
아래가 더 효율적인 방법 아닐까요~? FixedStrategy를 없애고 클래스 변수로 빼도 될 것 같네요.

```suggestion
		MoveStrategy strategy = () -> 4;
```

Reply:
제안해주신 방법이 테스트를 하기에도 더 용이한 측면이 있을것 같아요!